src/gui/screen_hardfault.cpp

```diff
@@ -25,4 +28,5 @@ screen_hardfault_data_t::screen_hardfault_data_t()
 void screen_hardfault_data_t::draw() {
     window_frame_t::draw();
     ScreenHardFault();
+    Sound_Play(eSOUND_TYPE::CriticalAlert);
```

Comment:
Are you sure that this method is not called periodically so that this sound is staring to play every time ?
CriticalAlert is repeating sound so integrity of this sound could be compromised.